gen3/bin/kube-setup-secrets.sh

```diff
@@ -28,6 +28,20 @@ if [[ -f "${WORKSPACE}/${vpc_name}/creds.json" ]]; then # update secrets
   cd "${WORKSPACE}"/${vpc_name}
 fi
 
+if [[ -f "${WORKSPACE}/${vpc_name}/creds.json" ]]; then # update secrets
+  if ! g3kubectl get secrets/aws-es-proxy > /dev/null 2>&1; then
+    credsFile=$(mktemp -p "$XDG_RUNTIME_DIR" "creds.json_XXXXXX")
+    creds=$(jq -r ".es|tostring" < creds.json |sed -e 's/[{-}]//g' -e 's/"//g' -e 's/:/=/g')
+    echo "[default]" > "$credsFile"
+    IFS=',' read -ra CREDS <<< "$creds"
+    for i in "${CREDS[@]}"; do
+      echo ${i} >> "$credsFile"
+    done
+    g3kubectl create secret generic aws-es-proxy "--from-file=credentials=${credsFile}"
+  fi
```

Comment:
Can we add the ES_PROXY endpoint to the secret too, then just mount that into the container rather than run `sed` on the deploy.yaml ?

Reply:
Also - we should probably leave a copy of the secret under `${vpc_name}/apis_configs/es/` - something like that?  It's not obvious to me where `creds.json` is coming from - maybe you meant to use `$credsFile` ?  I can stop by to have you explain it to me :-p